go-controller/pkg/ovn/egressip.go

```diff
@@ -257,15 +257,6 @@ func (oc *Controller) assignEgressIPs(eIP *egressipv1.EgressIP) error {
 	for _, egressIP := range eIP.Spec.EgressIPs {
 		klog.V(5).Infof("Will attempt assignment for egress IP: %s", egressIP)
 		eIPC := net.ParseIP(egressIP)
-		if eIPC == nil {
-			eIPRef := kapi.ObjectReference{
-				Kind: "EgressIP",
-				Name: eIP.Name,
-			}
-			oc.recorder.Eventf(&eIPRef, kapi.EventTypeWarning, "InvalidEgressIP", "egress IP: %s for object EgressIP: %s is not a valid IP address", egressIP, eIP.Name)
-			klog.Errorf("Unable to parse provided EgressIP: %s, invalid", egressIP)
-			continue
```

Comment:
Remove the Event but we should keep the check/Errorf/continue.

If something does go wrong somehow, we don't want ovn-kubernetes to end up crashlooping

Reply:
Wouldn't we in such a case, actually want to CrashLoop and investigate and understand what the heck happened in such a case?

Reply:
If this was CNO, maybe, since the result of it crashlooping is just "updates to network configuration stop propagating / cluster updates are blocked".

But ovn-kubernetes crashlooping means the entire cluster goes down.